mne/viz/evoked.py

```diff
@@ -185,7 +185,7 @@ def _plot_evoked(evoked, picks, exclude, unit, show,
     titles = _handle_default('titles', titles)
     units = _handle_default('units', units)
     # Valid data types ordered for consistency
-    channel_types = ['eeg', 'grad', 'mag', 'seeg', 'dipole', 'gof']
+    channel_types = ['eeg', 'grad', 'mag', 'eog', 'seeg', 'dipole', 'gof']
```

Comment:
I think this line determines which types get auto-plotted @agramfort, it's what I needed to change to get this plotting function to auto-plot the `DipoleFixed` data